climpred/reference.py

```diff
@@ -89,7 +89,8 @@ def compute_persistence(
     # at lead 0 is == 1.
     if [0] in hind.lead.values:
         hind = hind.copy()
-        hind['lead'] += 1
+        with xr.set_options(keep_attrs=True):
```

Comment:
First time I use this :)

Reply:
I never get it to work consistently, but I'm glad it does here!

Reply:
Can you add a small comment as to what this fixes? What attributes were being dropped?